### ISSUE TITLE
Size, Bandwidth and Remaining Time always empty when using --log-file option

### ISSUE THREAD

**Issue by ioogithub** [kapitainsky/RcloneBrowser#142]
Others users have requested that --log-file can be used as an option. I have been testing and I do not believe this is a feature request so much as a bug in parsing the output from rclone. Here is what I found:

1. If the option "--log-file" is used with no other options then the GUI is 100% broken, all of the progress fields are empty.

2. If the options "--log-file" and "--progress -vv" are used then *some* of the GUI will work:

Size: Always empty
Elapsed Time: works
Bandwidth: Always empty
Transfers: works
Total Size: works
Remaining Time: Always empty
Errors: always 0, this is another bug
Checks: works

The information is available however in the "Show Output" box so to fix the bug, Size, Bandwidth and Remaining Time would need to be properly parsed from rclone output.

**Comment by kapitainsky** [comment]
Thank you for this testing - this indeed points me into promising solution direction.

**Comment by ioogithub** [comment]
I found another bug. The GUI is not parsing the file progress correctly either.

For example, I have transfers set to 3 but only 2 files show as transferring. The output stream is truncating the last filename. If I scroll up however to the next instance of where the progress was outputted, I can see all 3 files transferring.

After the first two files complete, the transfer progress get stuck at 99% it doesn't update to 100% and remove them from the list so they collect on the screen at 99%

I had a test transfer for a while and it is now showing 35 files transferring at 99% which is incorrect.

I would say the file transfer progress is also broken when using the --log-file and --progress options.

The show output stream is always correct however, it shows 3 files being transferred.

**Comment by kapitainsky** [comment]
yes I am aware of it - this is due to changes in rclone output I have not included in my parser yet
